patches/chrome-browser-resources-settings-privacy_page-privacy_page.html.patch

```diff
@@ -62,10 +92,11 @@ index 9b9618c956d3ae257a147e4061ca18ede06fbf2b..37bb2d25092833a358a72ea0e027511e
            <category-setting-exceptions
                category="{{ContentSettingsTypes.JAVASCRIPT}}"
                block-header="$i18n{siteSettingsBlock}"
-@@ -445,6 +453,19 @@
+@@ -484,6 +501,21 @@
            </category-setting-exceptions>
          </settings-subpage>
        </template>
++<if expr="not _google_chrome">
 +      <template is="dom-if" route-path="/content/autoplay" no-search>
```

Comment:
this should be defined in the import `personalization_options` so it can be reduced to one line

Reply:
having some trouble getting this to work so pushing to a follow-up